test/e2e/page-objects/OIEPrimaryAuthPage.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2015-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const FormPage = require('./FormPage');
+
+class OIEPrimaryAuthPage extends FormPage {
+
+  constructor () {
+    super();
+    this.identifierUsernameInput = element(by.css('[id="okta-signin-username"]'));
+    this.credentialPasscodeInput = element(by.css('[id="okta-signin-password"]'));
```

Comment:
how come this would work? we don't have those IDs for OIE login form.

Reply:
I realized that these tests are running in interop mode i.e. running /v1/authn pipeline against OIE enabled org. So it'll still render the v1 login form. I don't need to add new page objects for this. 🤦 
I still think that we should have the same specs that we run against v1 org, run against an OIE enabled org as well to test backward compatibility.

Reply:
Make sense, so we'll have ?
- `/authn` against classical org
- `/authn` against OIE org
- `/idx` against OIE org

Reply:
Correct. I have already filed a bug for the 2nd case (/authn against OIE org) just by running these tests - https://oktainc.atlassian.net/browse/OKTA-375434
Next would be to run a subset of these tests + new E2E tests against /idx endpoints.